### PR TITLE
{tools}[iomkl/2018a] Subversion 1.9.7 with dependencies

### PR DESCRIPTION
**Pull request by branfosj**
Dependencies: 

- APR-util 1.6.1
- APR 1.6.3
- SCons 3.0.1
- Serf 1.3.9

### CODE REVIEW

File: easybuild/easyconfigs/t/Tk/Tk-8.6.8-iomkl-2018a-no-X11.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.8'
+versionsuffix = '-no-X11'
```
Comment (boegel) [inline easybuild/easyconfigs/t/Tk/Tk-8.6.8-iomkl-2018a-no-X11.eb]: @branfosj Why is this easyconfig included in here? It's not required for the dependencies, and the `Tk-8.6.8-iomkl-2018a.eb` we already have should be preferred anyway...

Can you please remove this easyconfig file from your PR?

**Comment by bartoldeman** [comment]
I wonder why GCCcore is not used here?

**Comment by boegel** [comment]
@bartoldeman Probably because this was modeled after the existing `Subversion` easyconfigs. Using `GCCcore` would indeed make more sense...

**Comment by branfosj** [comment]
@boegel I've removed Tk. Not sure how that made it into the pull request.

**Comment by boegel** [comment]
@bartoldeman Actually, it's a bit more complex than just switching the toolchain to `GCCcore`, since `SCons` needs Python as a dependency...

I'll go ahead and get this merged (if the tests work out), but for future updates of `Subversion` & `SCons` we should indeed look into lowering this to the appropriate subtoolchain.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2430.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5520e932156b22a5d1a92759afad854c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**